R/control.ergm.R

```diff
@@ -431,6 +447,8 @@ control.ergm<-function(drop=TRUE,
                        MPLE.type=c("glm", "penalized"),
                        MPLE.nonident=c("warning","message","error"),
                        MPLE.nonident.tol=1e-10,
+                       MPLE.covariance.samplesize = 500,
+                       MPLE.covariance.method = c("invHess",  "Godambe", "Bootstrap"),
```

Comment:
Please add this to the `match.arg.pars` below to check for validity and enable partial matching. Also, "Bootstrap" is not a proper noun.

Reply:
I have added "MPLE.covariance.methods" to match.arg.pars. I have also renamed "Bootstrap" to "bootstrap".